src/components/text-banner/text-banner.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styles from './text-banner.module.scss';
+import classNames from 'classnames';
+
+interface TextBannerProps {
+    title: string;
+    children?: React.ReactNode;
+    subtitle?: string;
+    className?: string;
+}
+
+export const TextBanner = ({ title, subtitle, className, children }: TextBannerProps) => {
```

Comment:
Not sure we need a component for this.

Could be just:
```jsx
<div className="textBanner">
    <div className="textBannerSubtitle">Subtitle</div>
    <div className="textBannerTitle">Title</div>
    <CategoryLink categorySlug="all-products">
        <LabelWithArrow>Shop Collections</LabelWithArrow>
    </CategoryLink>
</div>
```

It's even fewer lines than using a component because you don't need the import statement.